spec/ddtrace/context_provider_spec.rb

```diff
@@ -137,12 +138,19 @@ def thread_contexts
     context 'given a thread' do
       subject(:local) { thread_local_context.local(thread) }
 
-      let(:thread) { Thread.new {} }
-
```

Comment:
I believe this removal [breaks the subject](https://app.circleci.com/pipelines/github/DataDog/dd-trace-rb/4305/workflows/dea6b510-15a4-45be-b308-7d6c6d214660/jobs/164101)